data-service/src/main/java/uk/gov/gchq/palisade/service/data/exception/SerialiserNotFoundException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.data.exception;
+
+/**
+ * A {@link RuntimeException} that is thrown when incorrect values are used for the service Prepopulation.
+ */
+public class SerialiserNotFoundException extends RuntimeException {
+
+    /**
+     * Constructs a new {@link SerialiserNotFoundException} with the specified detail message and cause.
+     *
+     * @param cause     a {@link Throwable} that caused the error
+     */
+    public SerialiserNotFoundException(final Throwable cause) {
+        super(cause);
+    }
+
+    /**
+     * Constructs a new {@link SerialiserNotFoundException} with the specified detail message and cause.
```

Comment:
change it so javadoc is specific for each constructor. 
One for message
One for cause
one for both, same in SerlaiserInitialisationException and SerialiserNotFoundException

Reply:
Fixed